app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -340,6 +340,15 @@ def validate(self, data):
                         }
                     )
 
+            if (
+                SiteFlag.objects.value(SiteFlagNameChoices.LAUNCHING_DISABLED)
+                and self.instance.status == NimbusExperiment.Status.DRAFT
```

Comment:
Request for launch is also possible from PREVIEW status - probably want to add a check for that status here too. 

This will also allow launch review approval in a case where launch was requested and then launch was disabled just after. That might be more of an edge case?